CMakeLists.txt

```diff
@@ -224,9 +224,15 @@ if(WITH_SPDK)
   if(NOT WITH_BLUESTORE)
     message(SEND_ERROR "Please enable WITH_BLUESTORE for using SPDK")
   endif()
-  include(BuildSPDK)
-  build_spdk()
-  set(HAVE_SPDK TRUE)
+  find_package(uuid REQUIRED)
+  set(HAVE_UUID ${UUID_FOUND})
+  if(HAVE_UUID TRUE)
+    include(BuildSPDK)
+    build_spdk()
+    set(HAVE_SPDK TRUE)
+  else
+   set(HAVE_SPDK FALSE)
```

Comment:
please do not set `HAVE_SPDK` to FALSE when `WITH_SPDK` is ONE. if user wants to build a ceph `WITH_SPDK`, we should do so or just fail.